packages/sidecar/vite.config.ts

```diff
@@ -14,6 +14,9 @@ export default defineConfig({
     ssr: './src/main.ts',
     outDir: './dist',
     sourcemap: true,
+    rollupOptions: {
+      external: [...dependencies, 'node:path', 'node:http', 'node:fs', 'node:zlib'],
```

Comment:
l: Instead of passing the explicit list here (and risking other builtins being bundled), we could use the [`builtinModules` API](https://nodejs.org/api/module.html#modulebuiltinmodules). We do this in the JS SDK repo. WDYT?

Reply:
Happy to give it a look, but in a follow up as this pattern is prevalent in the spotlight repo.

Reply:
#441